buildpack/telemetry/metrics.py

```diff
@@ -176,25 +176,41 @@ def get_influx_registry():
 
 
 def get_statsd_registry():
+    allowed_metrics = os.getenv("APM_METRICS_FILTER_ALLOW")
```

Comment:
Let's keep this allow and deny logic outside of `get_statsd_registry` method, as it is not tied to the registry functionality and can be re-used if needed.

Reply:
isolated the logic in a separate method now. get_statsd_registry can be called with empty lists if we need to use it anywhere else without filters